doc/user/content/ops/troubleshooting.md

```diff
@@ -47,30 +37,24 @@ results than you expect, you can identify which operators
 take the largest total amount of time.
 
 ```sql
--- Extract raw elapsed time information, by worker
-SELECT mdo.id, mdo.name, mdo.worker_id, mse.elapsed_ns
+-- Extract raw elapsed time information, summed across workers
+SELECT mdo.id, mdo.name, mse.elapsed_ns
 FROM mz_internal.mz_scheduling_elapsed AS mse,
      mz_internal.mz_dataflow_operators AS mdo
-WHERE
-    mse.id = mdo.id AND
-    mse.worker_id = mdo.worker_id
+WHERE mse.id = mdo.id
 ORDER BY elapsed_ns DESC;
 ```
 
 ```sql
--- Extract raw elapsed time information, summed across workers
-SELECT mdo.id, mdo.name, sum(mse.elapsed_ns) AS elapsed_ns
-FROM mz_internal.mz_scheduling_elapsed AS mse,
+-- Extract raw elapsed time information, by worker
+SELECT mdo.id, mdo.name, mse.worker_id, mse.elapsed_ns
+FROM mz_internal.mz_scheduling_elapsed_per_worker AS mse,
      mz_internal.mz_dataflow_operators AS mdo
-WHERE
-    mse.id = mdo.id AND
-    mse.worker_id = mdo.worker_id
-GROUP BY mdo.id, mdo.name
+WHERE mse.id = mdo.id
 ORDER BY elapsed_ns DESC;
 ```
 
-<!-- mz_raw_compute_operator_durations is not available yet. -->
-<!-- ### Why is Materialize unresponsive for seconds at a time?
```

Comment:
I am not entirely sure why this section was commented out, but the query works (after adjusting) so it's probably fine to include it again. (cc @morsapaes)